presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -7,6 +7,18 @@ queries. The CLI is a
 `self-executing <http://skife.org/java/unix/2011/06/20/really_executable_jars.html>`_
 JAR file, which means it acts like a normal UNIX executable.
 
+.. contents::
```

Comment:
I think this is overkill since this fits on one page. There are plenty of longer documents that don't have this. Eventually, we'll update the style to have a right side nav.